Gutenberg/cocoapods_helpers.rb

```diff
@@ -68,23 +68,25 @@ def gutenberg_pod(config: GUTENBERG_CONFIG)
     options = { path: File.exist?(local_gutenberg) ? local_gutenberg : DEFAULT_GUTENBERG_LOCATION }
 
     raise "Could not find Gutenberg pod at #{options[:path]}. You can configure the path using the #{local_gutenberg_key} environment variable." unless File.exist?(options[:path])
-  else
-    options[:git] = "https://github.com/#{GITHUB_ORG}/#{REPO_NAME}.git"
-    options[:submodules] = true
-  end
 
-  pod 'Gutenberg', options
-  pod 'RNTAztecView', options
+    pod 'Gutenberg', options
+    pod 'RNTAztecView', options
 
-  gutenberg_dependencies(options: options)
+    gutenberg_dependencies(options: options)
+  elsif options[:tag]
+    pod 'Gutenberg', podspec: "https://cdn.a8c-ci.services/gutenberg-mobile/Gutenberg-#{options[:tag]}.podspec"
+  elsif options[:commit]
+    pod 'Gutenberg', podspec: "https://cdn.a8c-ci.services/gutenberg-mobile/Gutenberg-#{options[:commit]}.podspec"
+  end
 end
 
 def gutenberg_dependencies(options:)
   if options[:path]
     podspec_prefix = options[:path]
+  elsif options[:tag] || options[:commit]
+    return # when referencing via a tag or commit, we download pre-built frameworks
```

Comment:
Nitpick: moving this return statement to the first line to make unexpected cases more prominent: 

```ruby
# when referencing via a tag or commit, we download pre-built frameworks
return if options.key?(:tag) || options.key?(:commit)

podspec_prefix = options[:path]
raise 'Unexpected Gutenberg dependencies configuration' if podspec_prefix.nil?

podspec_prefix += '/third-party-podspecs'
...
```

Reply:
Done in 90ea9004ba35da1acfc8c4d96c5cbba68860aae8